app/assets/javascripts/cardboard/new_tab_links.js

```diff
@@ -0,0 +1,10 @@
+$(document).on("page:update", function() {
+  $("a").click(function() {
+    var link_host = this.href.split('/')[3];
```

Comment:
@joeyjoejoejr This is another instance of the point discussed below; I'm going to change the variable name to be less confusing, but leave the actual `this.href. . .` alone.